test/adaptive_load/input_variable_setter_test.cc

```diff
@@ -0,0 +1,57 @@
+#include "external/envoy/source/common/config/utility.h"
+
+#include "adaptive_load/input_variable_setter_impl.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
+
+TEST(RequestsPerSecondInputVariableSetterConfigFactoryTest, GeneratesEmptyConfigProto) {
+  InputVariableSetterConfigFactory& config_factory =
+      Envoy::Config::Utility::getAndCheckFactoryByName<InputVariableSetterConfigFactory>(
+          "nighthawk.rps");
+
+  Envoy::ProtobufTypes::MessagePtr message = config_factory.createEmptyConfigProto();
+
```

Comment:
superfluous newlines? (personally I'm fine with this as is -- but at some point it was encouraged and we reached consensus to avoid being liberal with newlines, which is what I think a lot of the other codebase is leaning towards. Though it might not be super consistent as we don't enforce it).

Reply:
Cleaned that up now.



I'll be on the lookout...